vite.config.ts

```diff
@@ -409,8 +402,8 @@ export default defineConfig(async () => {
 
 					// For the mobile app build, we currently can't load
 					// cross-origin requests, so we want to essentially make
-					// just one big JS chunk.
-					if (gjOpts.platform === 'mobile') {
+					// just one big JS chunk. SSR also works better this way.
+					if (['mobile', 'ssr'].includes(gjOpts.platform)) {
```

Comment:
what does removing ssr here do?

Reply:
SSR builds now were outputting multiple files, which was screwing things up, so I'm putting it back to how we had it now where it just outputs a single file like the mobile app build does. I'm sure I can get this working if I spend enough time, but I don't want to waste time doing all these minor things